### PR TITLE
remove (large) dependency on momentjs

### PR DESCRIPTION
**Pull request by nelix**
Don't merge yet, its poorly tested.

I thought I'd have a crack at removing momentjs, it was pretty easy.
Fecha is not a great library, but its pretty great at the format part. Open to other suggestions for libraries that are not 30kb.

### CODE REVIEW

File: packages/comments/comment.jsx [hunk 1]
```diff
@@ -167,7 +166,7 @@ module.exports = React.createClass({
 
     let editedDisplay = null;
 
-    if (this.props.comment.updatedAt && moment(this.props.comment.updatedAt).isAfter(moment(this.props.comment.createdAt))) {
+    if (this.props.comment.updatedAt && (this.props.comment.updatedAt > this.props.comment.createdAt)) {
```
Comment (grapegravity) [inline packages/comments/comment.jsx]: This might not be a safe assumption to make, as it does not take into account time zones or formats.

Comment (nelix) [inline packages/comments/comment.jsx]: anyone who sends time from their server in not UTC is asking for trouble.

Comment (deoxxa) [inline packages/comments/comment.jsx]: +1

Comment (nelix) [inline packages/comments/comment.jsx]: I'm not really interested in catering to bad practices, I'm open to warning somehow in theory...

Comment (grapegravity) [inline packages/comments/comment.jsx]: I mean, totally, but this is for public consumption and we might not want to make that assumption. Given we’re willing to drop Moment’s parsing, we can probably get away with at least trying to compare real Date objects, right?

Comment (nelix) [inline packages/comments/comment.jsx]: Seems legit

**Comment by nelix** [comment]
updated PR

**Pull request merged**